src/vmm/src/guest_config/templates/mod.rs

```diff
@@ -394,6 +396,10 @@ mod tests {
     #[cfg(target_arch = "x86_64")]
     use crate::guest_config::x86_64::X86_64CpuConfiguration;
 
+    const TEMPLATE_DESERIALIZATION_MAX_US: u128 = 200;
```

Comment:
where are these numbers coming from?

Reply:
They are arbitrary, as long as they were not in the 100s of millseconds I was happy. This testing in the unit test won't be long term though.